integration_test/screenshots_test.dart

```diff
@@ -0,0 +1,210 @@
+import 'dart:io';
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:font_awesome_flutter/font_awesome_flutter.dart';
+import 'package:integration_test/integration_test.dart';
+import 'package:timecop/blocs/locale/locale_bloc.dart';
+import 'package:timecop/blocs/notifications/notifications_bloc.dart';
+import 'package:timecop/blocs/projects/projects_bloc.dart';
+import 'package:timecop/blocs/settings/bloc.dart';
+import 'package:timecop/blocs/theme/theme_bloc.dart';
+import 'package:timecop/blocs/timers/timers_bloc.dart';
+import 'package:timecop/data_providers/data/data_provider.dart';
+import 'package:timecop/data_providers/data/mock_data_provider.dart';
+import 'package:timecop/data_providers/notifications/notifications_provider.dart';
+import 'package:timecop/data_providers/settings/mock_settings_provider.dart';
+import 'package:timecop/data_providers/settings/settings_provider.dart';
+import 'package:timecop/main.dart' as app;
+import 'package:timecop/screens/dashboard/components/RunningTimerRow.dart';
+import 'package:timecop/screens/dashboard/components/StoppedTimerRow.dart';
+
+bool didConvertedFlutterToSurfaceImage = false;
```

Comment:
As a bit of a grammar nazi, could you change the name of this variable to "convertedFlutterToSurfaceImage", please? :)

Reply:
Done.